imagescripts/common.sh

```diff
@@ -76,16 +77,14 @@ extract_database_url() {
       local jdbc_url="jdbc:mysql://$host_port_name?autoReconnect=true&characterEncoding=utf8&useUnicode=true&sessionVariables=default_storage_engine%3DInnoDB"
       local hibernate_dialect="org.hibernate.dialect.MySQLDialect"
       local database_type="mysql"
+      local validation_query="select 1"
       ;;
     sqlserver)
-      if [ -z "$(read_var $prefix PORT)" ]; then
-        eval "${prefix}_PORT=1433"
-      fi
-      local host_port_name="$(read_var $prefix HOST):$(read_var $prefix PORT)/$(read_var $prefix NAME)"
       local jdbc_driver="com.microsoft.sqlserver.jdbc.SQLServerDriver"
-      local jdbc_url="jdbc:sqlserver://$host_port_name"
+      local jdbc_url="jdbc:$url"
```

Comment:
Note that here we're basically just passing the parsed host/port/attributes downstream for `sqlserver` URLs